### PR TITLE
 Load weeks' content asynchronously #29 

### LINKED ISSUE #29: Load weeks' content asynchronously
The weeks loading is done in a non-asynchronous manner.
This effectively disables the page until everything is loaded on initial load as the JS thread is blocked (preventing the reveal animation for the contents).

### PR DESCRIPTION
**Pull request by benjaminheng**
Fixes #29 

- Load content asynchronously. Ajax function is wrapped so that the variable `week` is preserved for the success callback.
- Fix path to spinner image.

For testing purposes, throttling network speed (via Chrome's devtools for instance) will better show the difference between synchronous and async.

### CODE REVIEW

**Comment by damithc** [comment]
Thanks @benjaminheng for the PR.
@wkurniawan07 can you review?

**Comment by wkurniawan07** [comment]
Sure, but give me awhile. I need to get familiar with the code base again :p

File: scripts/common.js [hunk 1]
```diff
@@ -97,6 +97,38 @@ function checkIfAllComponentsChecked() {
...
+                var component = components[i];
+                makeAccordion('.' + component + '-week' + week);
+            }
+            for (var i in components) {
+                var component = components[i];
+                addCollapseAndExpandButtonsForComponents('#' + component + '-content-week' + week, component + '-week' + week);
+            }
+            for (var i in components) {
+                var component = components[i];
+            }
```
Comment (wkurniawan07) [inline scripts/common.js]: You can combine all the for loops above I guess, they're looping over the same variable.

**Comment by wkurniawan07** [comment]
Everything else looks good, just the minor suggestion above.

**Comment by damithc** [comment]
@benjaminheng when the PR is ready to review again, add a comment here to say so.

**Comment by benjaminheng** [comment]
Yup, it's ready.

**Comment by wkurniawan07** [comment]
Ok, good to merge :+1:

**Comment by damithc** [comment]
Thanks for the contribution @benjaminheng  :-)
Feel free to add your name to CONTRIBUTORS.md

**Pull request merged**